src/api/subscriptions.js

```diff
@@ -134,6 +134,13 @@ export class Subscriptions {
    */
   setOnLinkComplete(callback) {}
 
+  /**
+   * TODO(chenshay): Figure out what type the promise returns.
```

Comment:
If we return any result in the promise, I think it should simply be the passthrough value from the `accountPromise` for convenience. That way API clients won't need to overnest.

Reply:
done